packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -225,6 +227,11 @@ export const TrackScreenDetailsTile = ({
   const { isEnabled: isRemixContestEnabled } = useFeatureFlag(
     FeatureFlags.REMIX_CONTEST
   )
+  const { data: events } = useEventsByEntityId(trackId, {
```

Comment:
since it'll be pretty common to just get the "latest event" can we make a helper hook for this. Just like `useRemixContest`, which internally sets the entityType, gets the first event that matches remix-contest?